themes/default/content/blog/pulumi-release-notes-68/index.md

```diff
@@ -0,0 +1,128 @@
+
+As a result, we launched a public preview of Update Plans, a new Pulumi feature which guarantees that operations shown in `pulumi preview` will run on `pulumi up`. Update Plans also help catch any unexpected changes that might happen between when you preview a change and when you apply that change. Update Plans work by saving the results of a `pulumi preview` to a plan file, which enables you to restrict subsequent `pulumi up` operations to only the actions saved in the plan file. This helps you ensure that what you saw in the `pulumi preview` is what will actually happen when you run `pulumi up`.
+
+Learn more in our [blog post on Update Plans]({{< relref "/blog/announcing-public-preview-update-plans" >}}) and in this [Update Plans GitHub issue](https://github.com/pulumi/pulumi/issues/2318).
+
+
+### Disable default provider
+
+While default providers are enabled by default, you can now [disable default providers]({{< relref
+"/docs/intro/concepts/config#special-configuration-options" >}}) on a per stack basis. Disabling default
+providers is a good idea if you want to ensure that your programs must be explicit about which provider they
+will use. For example, to disable the `aws` provider, you can run:
+
+```sh
+$ pulumi config set --path 'pulumi:disable-default-providers[0]' aws
+```
+
+If you wanted to also disable the `kubernetes` default provider, as well as the `aws` default provider, you could run:
+
+```sh
+$ pulumi config set --path 'pulumi:disable-default-providers[1]' kubernetes
+```
+
+This adds a new entry to the list `pulumi:disable-default-providers`. To disable all default providers, use `*` as the package name:
+
+```sh
+$ pulumi config set --path 'pulumi:disable-default-providers[0]' '*'
+```
+Learn more in the [Disable Default Providers GitHub issue](https://github.com/pulumi/pulumi/issues/3383).
+
+### `pulumi up --target` wildcard URN support
+
+Our community had requested that we add wildcard URN support to `pulumi up --target` and as a result we now support wildcards for `pulumi up --target <urn>` and similar commands. Learn more in the [Wildcards support GitHub issue](https://github.com/pulumi/pulumi/issues/5870).
+
+## Pulumi Service & Pulumi.com
+
+### Sign-up and sign-in experience revamp
+
+We spent time this milestone revamping our sign-in and sign-up process in the Pulumi Service. Our intentions were to make it easier to differentiate and toggle between the sign-in and sign-up experiences. As part of this work we also refreshed the design to align with our public website. Take a look! 
```

Comment:
Might not focus on time spent - just note that we delivered improved UX on these important pages.